### DISCUSSION TITLE
just a question

### DISCUSSION THREAD

**Discussion by lis0** [maximmax42/Discord-CustomRP#32] [Q&A] [answered]
what are the Pipe window for?

**Comment by maximmax42** [comment, marked as answer]
It's for when you have more than 1 Discord client running at the same time. If you wanna change what account will the presence show up in, you would change the number, typically to 1.
Btw if the issue is closed you can still comment. Unless you can't, then plz tell me about it.

**Comment by lis0** [comment]
ahaa ok
